applications/DeepSpeed-Chat/training/step1_supervised_finetuning/main.py

```diff
@@ -145,6 +145,9 @@ def parse_args():
     parser.add_argument('--offload',
                         action='store_true',
                         help='Enable ZeRO Offload techniques.')
+    parser.add_argument('--bf16',
```

Comment:
Maybe try something like the following, so it can be easily expanded to other data types (e.g., fp8):
```python
  parser.add_argument('--dtype', type=str, default='fp16', choices=['fp16', 'bf16'], help = 'Training data type')
```

Reply:
I will follow your suggestions, thanks.